packages/web-features-figuring/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "web-features-figuring",
```

To avoid accidentally publishing this, which could well happen since it sits next to a package we will publish.

```suggestion
{
  "private": true,
  "name": "web-features-figuring",
```